l10n_it_ddt/models/stock_picking_package_preparation.py

```diff
@@ -79,7 +83,7 @@ class StockPickingPackagePreparation(models.Model):
 
     _inherit = 'stock.picking.package.preparation'
     _rec_name = 'display_name'
-    _order = 'date desc'
+    _order = 'ddt_number, date desc'
```

Comment:
Se non ricordo male l'ordinamento di default quando non specificato esplicitamente è asc.
Credo quindi in questo caso sia meglio scrivere: 
_order = 'ddt_number desc, date desc'

Reply:
:+1: infatti sulla 8 è `ddt_number desc`